sources/host-ctr/cmd/host-ctr/main.go

```diff
@@ -763,6 +763,30 @@ func withPrivilegedMounts() oci.SpecOpts {
 	})
 }
 
+// withStorageMounts adds options to share container storage mounts
+func withStorageMounts() oci.SpecOpts {
+	var mounts []runtimespec.Mount
+
+	storageDirs := []string{
+		"/var/lib/containerd",
+		"/var/lib/docker",
+		"/var/lib/kubelet",
+	}
+
+	for _, storageDir := range storageDirs {
+		if _, err := os.Stat(storageDir); !os.IsNotExist(err) {
+			mounts = append(mounts, runtimespec.Mount{
+				Options:     []string{"rbind", "rshared"},
+				Destination: fmt.Sprintf("/.bottlerocket/rootfs/%s", storageDir),
+				Source:      storageDir,
+				Type:        "bind",
+			})
+		}
+	}
+
+	return oci.WithMounts(mounts)
```

Comment:
nit: consistency with the other `with*Mounts` functions in the file:

```suggestion
	return withMounts(mounts)
```

Reply:
It felt unnecessary to call that function since all the mounts are known to have propagation options specified.

I could add a comment instead?

Reply:
Yeah, a comment will work :+1: